src/utilities/constants.js

```diff
@@ -15,3 +15,7 @@ export const MILLISECONDS = {
 export const SECONDS = {
   ONE_MINUTE: 60,
 };
+
+export const DATE_FORMAT = {
```

Comment:
Completely not necessary.. it was troubling me slightly that we had a string value all over the place for this one.

Reply:
I like it :)